src/overlays/actors/ovl_Obj_Ending/z_obj_ending.h

```diff
@@ -6,14 +6,14 @@
 struct ObjEnding;
 
 typedef struct {
-    /* 0x0 */   UNK_TYPE4 unk0;
-    /* 0x4 */   Gfx* dl;
+    /* 0x0 */   Gfx* dl1;
+    /* 0x4 */   Gfx* dl2;
     /* 0x8 */   AnimatedMaterial *texture;
-} unkStruct; // size = 0xC
+} ObjEndingGraphics; // size = 0xC
```

Comment:
unrelated but a better name for this might be `EndingModelInfo` or something, also `texture` should be `animMat`